nsone/rest/data.py

```diff
@@ -5,6 +5,20 @@
 #
 from . import resource
 
+class Job(resource.BaseResource):
```

Comment:
Yeah, sounds good to me, I'll get it done when I'm back home, probably in a
couple of hours.
On Jun 13, 2016 4:36 PM, "Pasha Palangpour" <notifications@github.com>
wrote:

> In nsone/rest/data.py
> <https://github.com/ns1/nsone-python/pull/6#discussion_r66791928>:
>
> > @@ -5,6 +5,20 @@
> >  #
> >  from . import resource
> >
> > +class Job(resource.BaseResource):
>
> Would you mind moving this class to the rest monitoring module
> <https://github.com/ns1/nsone-python/blob/master/nsone/rest/monitoring.py>
> ?
>
> And what if we called this class Monitors as opposed to Job to keep with
> the convention of Records
> <https://github.com/ns1/nsone-python/blob/master/nsone/rest/records.py#L9>
> and Zones
> <https://github.com/ns1/nsone-python/blob/master/nsone/rest/zones.py#L10>?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ns1/nsone-python/pull/6/files/e9267fab90850c84ad6a25e9c3b027ce710a9c2f#r66791928>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AL5VfbiZH-shN4gQ5X3qnhSsry7xxPxQks5qLVzLgaJpZM4I0IES>
> .
>